frontend/src/metabase/writeback/components/ActionCreator/ActionCreator.tsx

```diff
@@ -86,34 +96,61 @@ function ActionCreatorComponent({
   const afterSave = (action: SavedCard) => {
     setQuestion(question.setCard(action));
     setTimeout(() => setShowSaveModal(false), 1000);
-    if (!actionId && action.action_id) {
-      setTimeout(() => push(`/action/${action.action_id}`), 1500);
-    }
+    onClose?.();
   };
 
   const handleClose = () => setShowSaveModal(false);
 
   const isNew = !actionId && !(question.card() as SavedCard).id;
 
+  const handleExampleClick = () => {
+    const sampleQuery =
+      "UPDATE my_table_name\nSET my_column_name = {{ my_new_value }}\nWHERE id = {{ my_primary_key }}";
```

Comment:
Random idea: maybe use a query that'd act on a sample database as an example? It'd be actually runnable, so it'd be easier to explore how things work 🤷‍♂️